Kickstarter-iOS/Views/Controllers/BackerDashboardViewController.swift

```diff
@@ -370,6 +370,8 @@ extension BackerDashboardViewController: UIGestureRecognizerDelegate {
 
 extension BackerDashboardViewController: TabBarControllerScrollable {
   func scrollToTop() {
+    //swiftlint:disable todo
+    //FIXME: crash because self.pageViewController is nil
```

Comment:
I've added this to `[RELEASE] Settings v2` card so we keep track of it.

Reply:
I also believe I know how to reproduce this bug..it happens when you're toggling between an account that is a backer and that isn't.

Steps to reproduce:
1. Login as backer
2. Logout
3. Login as non-backer
>> should crash because `RootTabBarController` is trying to call `scrollToTop()` on `BackerDashboardViewController`